src/witan/app/data.clj

```diff
@@ -88,19 +92,21 @@
     (first (c/exec (find-data-by-data-id id)))))
 
 (defn data-to-db
-  [{:keys [data-id category name publisher version file-name s3-key public?] :as data}]
+  [{:keys [data-id file-name s3-key public? publisher-name] :as data}]
   (-> data
       (dissoc :data-id
```

Comment:
The `dissoc` is redundant, since the later `assoc` will (effectively) overwrite any values.